templates/registration/totp_enable.html

```diff
@@ -61,6 +65,52 @@
             }
         });
     </script>
+    <script src="{{ static('libs/clipboard/clipboard.js') }}"></script>
+    <script src="{{ static('libs/clipboard/tooltip.js') }}"></script>
+    <script type="text/javascript">
+        $(function () {
+            var info_float = $('.info-float');
+            if (info_float.length) {
+                var container = $('#content-right');
+                if (window.bad_browser) {
+                    container.css('float', 'right');
+                } else if (!featureTest('position', 'sticky')) {
+                    fix_div(info_float, 55);
+                    $(window).resize(function () {
+                        info_float.width(container.width());
+                    });
+                    info_float.width(container.width());
+                }
+            }
+
+            var copyButton;
+            $('pre code').each(function () {
+                $(this).parent().before($('<div>', {'class': 'copy-clipboard'})
+                    .append(copyButton = $('<span>', {
+                        'class': 'btn-clipboard',
+                        'data-clipboard-text': $(this).text(),
+                        'title': 'Click to copy'
+                    }).text('Copy')));
+
+                $(copyButton.get(0)).mouseleave(function () {
+                    $(this).attr('class', 'btn-clipboard');
+                    $(this).removeAttr('aria-label');
+                });
+
+                var curClipboard = new Clipboard(copyButton.get(0));
+
+                curClipboard.on('success', function (e) {
```

Comment:
```suggestion
                curClipboard.on('success', function(e) {
```

Only no-arg functions have a space before the `(`).